python/servo/build_commands.py

```diff
@@ -394,6 +398,45 @@ def build(self, target=None, release=False, dev=False, jobs=None,
             if not os.path.exists(aar_out_dir):
                 os.makedirs(aar_out_dir)
             env["AAR_OUT_DIR"] = aar_out_dir
+            # GStreamer and its dependencies use pkg-config and this flag is required
+            # to make it work in a cross-compilation context.
+            env["PKG_CONFIG_ALLOW_CROSS"] = '1'
+            # Build the name of the package containing all GStreamer dependencies
+            # according to the build target.
+            gst_lib = None
+            if re.match("arm-([a-z])*-androideabi", target):
+                gst_lib = "gst-build-armeabi"
+            elif re.match("armv7-([a-z])*-androideabi", target):
+                gst_lib = "gst-build-armeabi-v7a"
+            elif re.match("x86_64-([a-z])*-android", target):
+                gst_lib = "gst-build-x86_64"
+            elif re.match("i686-([a-z])*-android", target):
+                gst_lib = "gst-build-x86"
```

Comment:
We should be able to use `gst_lib = "gst-build-{}".format(self.config["android"]["lib"])` instead of this conditional.